contracts_lib_py/didregistry.py

```diff
@@ -498,6 +497,45 @@ def get_provenance_method_events(self, method, did_bytes):
                 f'did {did} at blockNumber')
         return result
 
+    def mint(self, did, amount, account):
+        tx_hash = self.send_transaction(
+            'mint',
+            (did, amount),
+            transact={'from': account.address,
+                      'passphrase': account.password,
+                      'keyfile': account.key_file}
+        )
+
+        return self.is_tx_successful(tx_hash)
+
+    def burn(self, did, amount, account):
+        tx_hash = self.send_transaction(
+            'burn',
+            (did, amount),
+            transact={'from': account.address,
+                      'passphrase': account.password,
+                      'keyfile': account.key_file}
+        )
+        return self.is_tx_successful(tx_hash)
+
+    def balance(self, address, did):
+        return self.contract.caller.balanceOf(address, did)
+
+    def transfer(self, did, address, amount, account):
```

Comment:
Maybe this should be renamed to something like `transfer_nft` so that this is not confused with transfer_ownership